pkg/cli/environment.go

```diff
@@ -76,6 +76,7 @@ type EnvConfigInstall struct {
 	Kind         string `yaml:"kind"`
 	Distribution string `yaml:"distribution"`
 	Namespace    string `yaml:"namespace"`
+	Verbose      bool   `yaml:"verbose"`
```

Comment:
This should not be a part of the EnvConfigInstall struct. Instead, it should be a part of the EnvSettings struct.